.github/workflows/e2e.yml

```diff
@@ -110,11 +110,12 @@ jobs:
       - name: Find Comment
         uses: peter-evans/find-comment@v2
         id: find_comment
+        if: ${{ github.event.pull_request.number }}
         with:
           issue-number: ${{ github.event.pull_request.number }}
           body-includes: "Found errors/fatal log records."
       - name: Create PR comment if errors/fatal logs produced
-        if: ${{ steps.check-logs.outputs.errors > 0 || steps.check-logs.outputs.fatal > 0 }}
+        if: $${{ github.event.pull_request.number }} && ({{ steps.check-logs.outputs.errors > 0 || steps.check-logs.outputs.fatal > 0 }})
```

Comment:
here is something wrong with the syntax. $$ should be $ and another $ should be added before steps.check

${{ github.event.pull_request.number }} && (${{ steps.check-logs.outputs.errors > 0 || steps.check-logs.outputs.fatal > 0 }})

Reply:
@mo-c4t I made the changes